packages/react-admin/src/Admin.js

```diff
@@ -44,7 +45,11 @@ const Admin = ({
         appReducer(action.type !== USER_LOGOUT ? state : undefined, action);
     const saga = function* rootSaga() {
         yield all(
-            [crudSaga(dataProvider, authClient), ...customSagas].map(fork)
+            [
+                crudSaga(dataProvider, authClient),
+                i18nSaga({ locale, messages, enableLocaleQuerySwitcher }),
```

Comment:
I'd prefer that you put the `i18nSaga` in the `crudSaga`, to avoid composing sagas here. Remember, the code of the `Admin` component becomes the code to duplicate for people who want to build a custom app. The less work they have, the better.